torch/nn/modules/rnn.py

```diff
@@ -296,9 +296,9 @@ class RNN(RNNBase):
 
     Attributes:
         weight_ih_l[k]: the learnable input-hidden weights of the k-th layer,
-            of shape `(input_size x hidden_size)`
+            of shape `(hidden_size x input_size)` (for k=0)
         weight_hh_l[k]: the learnable hidden-hidden weights of the k-th layer,
-            of shape `(hidden_size x hidden_size)`
+            of shape `(hidden_size x hidden_size)` (for k>0)
```

Comment:
This `for k` casing only applies to `weight_ih_l` where for `k = 0` it's `hidden x input` and then `hidden x hidden`. `weight_hh_l` is always `hidden x hidden`

Reply:
Fixed, thanks for correction!